clusters/hive/pools/pipelines/pipelines-install-config.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: pipelines-install-config
+  namespace: pipelines-cluster-pool
+type: Opaque
+stringData:
+  install-config.yaml: |-
+    apiVersion: v1
+    baseDomain: psi.ospqa.com
+    compute:
+    - name: worker
+      platform:
+        openstack:
+          type: ci.m1.xlarge
+      replicas: 2
```

Comment:
is this the minimum resource requirement  for master and worker

Reply:
@savitaashture I have a very limited idea on the resource requirements. Therefore, I haven't changed anything related to the resources. I have copied the configuration from the existing configuration used by the QE folks.
I am planning to change this based on the requirements. Please guide me if you think this needs to be changed.

Reply:
We will need more for the Performance team.

Reply:
@khrm do you know how much that should be?